<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>MOVE</string>
    </property>
    <widget class="QPushButton" name="moveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                               }</string>
     </property>
     <property name="text">
      <string>Move model</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dx_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dy_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dz_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>dz</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                               }</string>
    </property>
    <property name="text">
     <string>Load model</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>ROTATE</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>fx</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fx_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fy_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fz_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                               }</string>
     </property>
     <property name="text">
      <string>Rotate model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>fy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>fz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>SCALE</string>
    </property>
    <widget class="QDoubleSpinBox" name="kx_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ky_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="kz_Box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>kx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ky</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>kz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scaleButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>117</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
                                  display: inline-block;
                                  font-size: 12px;
                                  cursor: pointer;
                                  text-align: center;
                                  text-decoration: none;
                                  outline: none;
                                  color: #fff;
                                  background-color: #4CAF50;
                                  border: none;
                                  border-radius: 10px;
                                  box-shadow: 0 9px #999;
                                }
QPushButton:hover {background-color: #3e8e41}
QPushButton:pressed {
                                  background-color: #3e8e41;
                                  box-shadow: 0 5px #666;
                                  transform: translateY(4px);
                               }</string>
     </property>
     <property name="text">
      <string>Scale model</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="model_name">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Model</string>
    </property>
    <widget class="QComboBox" name="combomodelBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QPushButton" name="add_camButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add camera</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combocameraBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set camera</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear scene</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
